### DISCUSSION TITLE
Is this dead?

### DISCUSSION THREAD

**Discussion by davidjeba** [cloudflare/production-saas#20] [General]
I'd like to discuss on some ideas on building a saas platform

**Comment by lancedouglas1** [comment]
@lukeed is this project dead?

**Comment by lukeed** [comment]
I no longer work at Cloudflare.

**Reply by Mecanik** [reply]
So what does this entail? You will no longer maintain the project? It would be good to clarify because many, many people would like to use it. One last note, if you did maintain it, pretty sure you would have loads of sponsors.

**Comment by jackrobbins1** [comment]
+1 I was excited to use a CMS on the cloudflare stack

**Comment by davidjeba** [comment]
Me too, there are tons of cms for web.
For mobile there's none...
Strapi is good but it's more of a middleware than a cms.

Cms on cloudflare edge not like wordpress but something like wix.com/velo a
no-code website frontend with firebase or cockroach db as backend would be
great

If it's in python stack it will attract a lot of developers as it's leading
the TIOBE index
